src/main/java/org/squiddev/cobalt/lib/BaseLib.java

```diff
@@ -217,15 +219,27 @@ public Varargs invoke(LuaState state, Varargs args) throws LuaError, UnwindThrow
 				case 6: // "print", // (...) -> void
 				{
 					return noUnwind(state, () -> {
-						LuaValue tostring = OperationHelper.getTable(state, state.getCurrentThread().getfenv(), valueOf("tostring"));
```

Comment:
Maybe just keep the current code but use the global environment instead? We don't actually use this in CC anywhere, so not too fussed about making it work perfectly.